vars/selfUnitTest.groovy

```diff
@@ -0,0 +1,112 @@
+/* groovylint-disable ParameterName */
+// vars/selfUnitTest.groovy
+
+/**
+ * selfUnitTest.groovy
+ *
+ * Runs self unit tests.
+ */
+
+
+void call() {
+    // Save and restore env.pragmas after testing
+    env_pragmas_original = env.pragmas
+    try {
+        _test_pragmas_env()
+    } finally {
+        env.pragmas = env_pragmas_original
+    }
+}
+
+void _test_pragmas_env() {
+    commit_message = '''Skip-build: true
```

Comment:
We should probably make this commit message look more _real_, with a title and at least one line of description.

Maybe one day we will enforce that commit pragmas are only looked at after a title and description and this test should be ready for that.  It also ensures that a commit message with a title and description (which will be the usual case) also passes this test.

Reply:
Yes, I can do that